### ISSUE TITLE
Remember me checkbox on sign in page lacks focus indicator

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#2704]
The focus indicator for the 'Remember Me' checkbox has been removed/disabled on our sign in page. It is present on the default WordPress page. We should restore it.

https://images.zenhubusercontent.com/9209369/4b0ce3c7-78a4-423e-86a3-0008695dff11/focus_indicator.mp4

![Screenshot from 2022-03-16 07-20-39.png](https://images.zenhubusercontent.com/6012398f80b7cb0f8d1385bc/0f3cd6a5-2c7d-4567-903a-d99b54fa6569)

Here's the javascript for the login page: https://github.com/pressbooks/pressbooks/blob/dev/assets/src/scripts/login.js

Here's the CSS for the login page: https://github.com/pressbooks/pressbooks/blob/dev/assets/src/styles/login.scss

https://github.com/pressbooks/pressbooks/blob/3eefe8a9e1cf0974241800acf72c6e6a6da65be7/assets/src/styles/login.scss#L53-L62
